hack/fetch-official-cve-feed.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import requests
+
+url = 'https://api.github.com/search/issues?q=is:issue+label:official-cve-feed+\
+state:closed+repo:kubernetes/kubernetes'
+headers = {'Accept': 'application/vnd.github.v3+json'}
+res = requests.get(url, headers=headers)
+cve_arr = res.json()
+
+cve_list = []
+
+for item in cve_arr['items']:
+    cve = {"issue_url": None, "number": None, "cve_id": None,
+           "summary": None, "cve_url": None, "google_group_url": None}
+    cve['issue_url'] = item['html_url']
+    cve['number'] = item['number']
+    title = item['title'].replace(" -", ":")
+    title = title.split(": ")
+    print(title)
+    print(len(title))
+    if len(title) == 1:
+        cve_id = None
+        cve['cve_id'] = None
+        cve['cve_url'] = None
+        cve['summary'] = title[0]
+        cve['google_group_url'] = None
+    else:
+        cve_id = title[0]
+        cve['cve_id'] = title[0]
+        if len(title) == 3:
+            cve['summary'] = title[2]
+        else:
+            cve['summary'] = title[1]
+        cve['cve_url'] = f"https://www.cvedetails.com/cve-details.php?t=1&cve_id={cve_id}"
```

Comment:
```suggestion
        cve['cve_url'] = f"https://www.cvedetails.com/cve-details.php?cve_id={cve_id}"
```

Reply:
`&` is getting replaced in the HTML and JSON parsing, so the above might be easier to parse and use as is

Reply:
Updated